packages/block-editor/src/components/block-list/style.scss

```diff
@@ -420,22 +420,23 @@
 			}
 		}
 
-		> .block-editor-block-mover .block-editor-block-mover__control {
+		> .block-editor-block-list__block-edit > .block-editor-block-mover .block-editor-block-mover__control {
 			float: left;
 		}
 
 		// Position hover label on the right
-		> .block-editor-block-list__breadcrumb {
+		> .block-editor-block-list__block-edit > .block-editor-block-list__breadcrumb {
 			right: -$border-width;
```

Comment:
Adding a `+1 px` here solves the hover issue for me

Reply:
> Adding a `+1 px` here solves the hover issue for me

I think it would effectively make it equivalent as `right: 0;` which _sounds_ like what it should be (and conversely, the negative value could certainly explain the overflow). FWIW, I'm not able to reproduce the issue in Chrome, so I might imagine it's browser-specific, and I wouldn't doubt it's an issue which has been present as well.

Reply:
I think this needs @kjellr input. Btw with the mover above, I think we could do with the hover label on the left again.